ui/pages/settings/networks-tab/network-form/network-form.component.js

```diff
@@ -433,23 +447,29 @@ export default class NetworkForm extends PureComponent {
 
   validateUrlRpcUrl = (url, stateKey) => {
     const { t } = this.context;
-    const { rpcUrls } = this.props;
+    const { networksToRender } = this.props;
     const { chainId: stateChainId } = this.state;
-    const isValidUrl = validUrl.isWebUri(url) && url !== '';
+    const isValidUrl = validUrl.isWebUri(url);
     const chainIdFetchFailed = this.hasError(
       'chainId',
       t('failedToFetchChainId'),
     );
+    const [matchingRPCUrl] = networksToRender.filter((e) => e.rpcUrl === url);
 
-    if (!isValidUrl) {
+    if (!isValidUrl && url !== '') {
```

Comment:
What does this change do for the error state of the field?

Reply:
so when url is an empty string, `validUrl.isWebUri(url)` is `undefined`. And so `const isValidUrl = validUrl.isWebUri(url) && url !== ''` would equal `undefined` when url is an empty string and `!isValidUrl` would equal `true`, causing an incorrect validation error "Invalid RPC URL" to show when the string is empty.